File: src/main/java/com/rs/java/game/area/multi/BurthorpeMulti.java
package com.rs.java.game.area.multi;

import com.rs.java.game.WorldTile;
import com.rs.java.game.area.Area;
import com.rs.java.game.area.shapes.Polygon;
import com.rs.java.game.area.shapes.Rectangle;

public class BurthorpeMulti extends Area {

	public BurthorpeMulti() {
		super(
				new Rectangle(new WorldTile(2880, 3520, 0), new WorldTile(2903, 3543, 0))

		);
	}

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Burthorpe Multi";
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}
}

File: src/main/java/com/rs/kotlin/game/world/area/areas/multi/BurthorpeMulti.kt
package com.rs.kotlin.game.world.area.areas.multi

import com.rs.java.game.WorldTile
import com.rs.kotlin.game.world.area.Area
import com.rs.kotlin.game.world.area.shape.Rectangle

class BurthorpeMulti : Area(
    Rectangle(WorldTile(2880, 3520, 0), WorldTile(2903, 3543, 0))

) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Burthorpe Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}